pkgs/by-name/li/linuxhw-edid-fetcher/package.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, coreutils
+, curl
+, fetchFromGitHub
+, gawk
+, gnutar
+, stdenv
+, unixtools
+, writeShellApplication
+, nix-update-script
+, displays ? { }
+}:
+
+# Usage:
+#   let
+#     edids = (linuxhw-edid-fetcher.override {
+#       displays = {
+#         PG278Q_2014 = [ "PG278Q" "2014" ];
+#       };
+#     });
+#   in
+#     "${edids}/lib/firmware/edid/PG278Q_2014.bin";
```

Comment:
```suggestion
#   let
#     edids = linuxhw-edid-fetcher.override {
#       displays.PG278Q_2014 = [ "PG278Q" "2014" ];
#     };
#   in
#     "${edids}/lib/firmware/edid/PG278Q_2014.bin";
```
we can minimize that example a bit further, to make it easier to understand

Reply:
actually I added (unnecessary) `2560x1440` to make it obvious `PG278Q_2014.bin` is derived from the object key, not a search path.